### ISSUE TITLE
Make the website responsive for smaller devices.

### ISSUE THREAD

**Issue by vishalpawarR** [ArshErgon/ResumeWizard#15]
@ArshErgon Hey On smaller Screen sizes the page is breaking.
![image](https://github.com/ArshErgon/ResumeWizard/assets/55725885/aab2b540-36eb-4905-bb42-e59c3a83cce4)

**Comment by ArshErgon** [comment]
yeah, its breaking down for mobile devices, as for now I only focusing on desktop version, because for mobile the editing and giving the proper suggestion would be quite different. A complete new site would be needed for mobile devices.

**Comment by vishalpawarR** [comment]
@ArshErgon Thank you for letting me know.

**Comment by ArshErgon** [comment]
gonna reopen this issue as my main resume is ready only the RightSideBar is left after that add more resume will be easy, then the main priority would be making the website suitable for mobile devices

**Comment by Tharsiga-21** [comment]
Hi can I work on this issue
